dagger-functions/src/test/resources/test_udf.py

```diff
@@ -0,0 +1,7 @@
+from pyflink.table import DataTypes
+from pyflink.table.udf import udf
+
+
+@udf(result_type=DataTypes.STRING())
+def test_function(text: str):
```

Comment:
is this not a Scalar function?

Reply:
@udf -> scalar function
@udtf -> table function
@udaf -> aggregate function